src/Card/CardStatus.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '..';
+
+const CardStatus = React.forwardRef(({
+  className,
+  children,
+  variant,
+  icon,
+}, ref) => (
+  <div
+    className={classNames('pgn__card-status',
+      `alert-${variant}`, 'alert', 'alert-content',
+      className)}
+    ref={ref}
+  >
+    {icon && <Icon src={icon} className="alert-icon" />}
+    {children}
+  </div>
+));
+
+CardStatus.propTypes = {
+  /** Specifies the content of the component. */
+  children: PropTypes.node,
+  /** The class to append to the base element. */
+  className: PropTypes.string,
+  /** Icon that will be shown in the alert */
+  icon: PropTypes.func,
+  /** Specifies variant to use. */
+  variant: PropTypes.oneOf([
+    'primary',
+    'secondary',
+    'success',
+    'danger',
+    'warning',
+    'info',
+    'light',
+    'dark',
```

Comment:
I'm wondering if we want to expose all these variants as options in the prop types if the design spec only calls for warning, success, danger, dark.